src/Webfactory/Slimdump/SlimdumpCommand.php

```diff
@@ -56,10 +58,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
     private function connect($dsn)
     {
         try {
-            return \Doctrine\DBAL\DriverManager::getConnection(
-                array('url' => $dsn, 'charset' => 'utf8', 'driverClass' => 'Doctrine\DBAL\Driver\PDOMySql\Driver')
+            /**
+             * a dsn string containing mysqli will result in the DriverManager creating a MysqliConnection but we need PDO
+             * @see DriverManager::parseDatabaseUrlScheme()
+             */
+            $mysqliIndependentDsn = str_replace('mysqli://', 'mysql://', $dsn);
+
+            return DriverManager::getConnection(
+                array('url' => $mysqliIndependentDsn, 'charset' => 'utf8', 'driverClass' => 'Doctrine\DBAL\Driver\PDOMySql\Driver')
```

Comment:
`'driverClass' => 'Doctrine\DBAL\Driver\PDOMySql\Driver'` is actually pointless, it will be overridden as well when the URL is parsed